README.md

```diff
@@ -1,8 +1,8 @@
 # Xunit Test Logger
 Xunit logger extensions for [Visual Studio Test Platform](https://gtihub.com/microsoft/vstest).
 
-[![Build Status](https://travis-ci.com/Nipunam/xunit.testlogger.svg?branch=master)](https://travis-ci.com/Nipunam/xunit.testlogger)
-[![Build status](https://ci.appveyor.com/api/projects/status/rac9mgpdslmffoqd?svg=true)](https://ci.appveyor.com/project/Nipunam/xunit-testlogger)
+[![Build Status](https://travis-ci.com/spekt/xunit.testlogger.svg?branch=master)](https://travis-ci.com/spekt/xunit.testlogger)
+[![Build status](https://ci.appveyor.com/api/projects/status/73iw12g89lhlr9ir?svg=true)](https://ci.appveyor.com/project/spekt/xunit-testlogger)
```

Comment:
- I think in line 18 and 24 `--test-adapter-path:.` not required any more. see if you can remove it.
-  replace `loggerFile.xml` with `test-results.xml`

Reply:
I will remove the `--test-adapter-path` after we have end to end tests. Right now we don't have validation in this branch.